isis/src/qisis/objs/AbstractProjectItemView/AbstractProjectItemView.cpp

```diff
@@ -45,11 +46,32 @@ namespace Isis {
   AbstractProjectItemView::AbstractProjectItemView(QWidget *parent) : QMainWindow(parent) {
 
     setWindowFlags(Qt::Widget);
+
+    QSizePolicy policy = sizePolicy();
+    policy.setHorizontalPolicy(QSizePolicy::Expanding);
+    policy.setVerticalPolicy(QSizePolicy::Expanding);
+    setSizePolicy(policy);
+
     m_internalModel = new ProjectItemProxyModel(this);
     setAcceptDrops(true);
   }
 
 
+  /**
+   * Returns the suggested size
+   *
+   * @return @b QSize The size hint
+   */
+  QSize AbstractProjectItemView::sizeHint() const {
+
+    //  Size hint is make ridiculously large as a hack to have the views fill the available dock
+    //  space.  SizePolicy alone did not work.
+    // TODO:  There should be a better way to do this.
```

Comment:
This isn't a code review comment, but since this is listed as a "TODO," is there a Trello card or note somewhere to find a better way to do this?(maybe as a stretch goal?)